### PR TITLE
RFC 6265bis: Add Lax-allowing-unsafe SameSite enforcement mode

### PR DESCRIPTION
**Pull request by chlily1**
Addresses #1422

### CODE REVIEW

**Comment by chlily1** [comment]
@englehardt PTAL.

Ended up adapting the text from Cookie Incrementalism into two sections, one with a focus on the behavior (with a monkey patch to the algorithm) and the other more focusing on the justification for it.

File: draft-ietf-httpbis-rfc6265bis.md
Comment (englehardt) [inline draft-ietf-httpbis-rfc6265bis.md:1324]: I think the wording here is a bit open ended. How about:

"To limit the scope of this compatibility mode, user agents which apply "Lax-allowing-unsafe" SHOULD restrict the enforcement to cookies which were created recently. Deployment experience has shown that a cookie age of 2 minutes or less to be a reasonable limit."

**Review by englehardt — APPROVED** [review]
Overall this LGTM. Having this as a monkey patch seems like the best solution. I've added two suggestions for alternate wording.

File: draft-ietf-httpbis-rfc6265bis.md
Comment (chlily1) [reply draft-ietf-httpbis-rfc6265bis.md]: Done.

File: draft-ietf-httpbis-rfc6265bis.md
Comment (chlily1) [reply draft-ietf-httpbis-rfc6265bis.md]: Done.

**Comment by chlily1** [comment]
Thanks for the review!
This is dependent on 6265-non-http-apis so I'll merge it after that one lands.

**Pull request merged**
